### PR TITLE
feat: preview experience

### PR DESCRIPTION
**Pull request by AdiGajbhiye**
## Checklist

- [ ] I have run this code and it appears to resolve the stated issue
- [ ] `README.md` updated and added information about my change

### CODE REVIEW

File: src/altimate.ts [hunk 1]
```diff
@@ -110,6 +110,23 @@ export class AltimateRequest {
...
+  handlePreviewFeatures(): boolean {
+    if (this.enabled()) {
+      return true;
+    }
+    window
+      .showInformationMessage(
+        `You need Altimate AI API Key to use this feature. Get your API Key for free by signing up`,
+        "Sign Up for API Key",
+      )
+      .then((answer) => {
```
Comment (mdesmet) [inline src/altimate.ts:122]: use await and async and use a enum or a constant for the option, there are examples.

File: package.json [hunk 2]
```diff
@@ -269,8 +269,12 @@
       },
       {
         "command": "dbtPowerUser.sqlToModel",
-        "title": "Convert to dbt Model",
-        "icon": "$(circuit-board)"
+        "title": "Convert to dbt Model(Preview feature)",
```
Comment (mdesmet) [inline package.json:272]:
```suggestion
        "title": "Convert to dbt Model (Preview feature)",
```

File: src/altimate.ts [hunk 3]
```diff
@@ -115,6 +119,24 @@ export class AltimateRequest {
     return !!this.getConfig();
   }
 
+  private async showAPIKeyMessage() {
+    const answer = await window.showInformationMessage(
+      `You need Altimate AI API Key to use this feature. Get your API Key for free by signing up`,
+      PromptAnswer.YES,
+    );
+    if (answer === PromptAnswer.YES) {
+      env.openExternal(Uri.parse("https://app.myaltimate.com/register"));
```
Comment (suryaiyer95) [inline src/altimate.ts:128]:
```suggestion
      env.openExternal(Uri.parse("https://app.myaltimate.com/register?source=extension"));
```

**Review by suryaiyer95 — COMMENTED** [review]
We need to add this for the query explanation as well. That piece is already merge into master

**Review by anandgupta42 — APPROVED** [review]
LGTM

**Pull request merged**